src/themes/dojo/titlePane.m.css

```diff
@@ -0,0 +1,34 @@
+@import './variables.css';
+
+.root {
+	overflow: hidden;
+	box-sizing: border-box;
+}
+
+.root * {
+	box-sizing: border-box;
+}
+
+.title {
+	background-color: var(--title-bar-bg);
+	border-color: var(--title-bar-bg);
+	border-style: solid;
+	border-width: var(--border-thin);
+	color: var(--title-bar-color);
+	font-size: var(--title-bar-font-size);
+	padding: var(--title-padding);
+	position: relative;
+	z-index: 1;
+}
+
+.closeable {
+	cursor: pointer;
```

Comment:
It is, but since the theme replaces the base CSS entirely, we also have to put it here.